scaffolding-chef/plan.ps1

```diff
@@ -0,0 +1,14 @@
+$pkg_name="scaffolding-chef"
+$pkg_description="Scaffolding for Chef Policyfiles"
+$pkg_origin="core"
+$pkg_version="0.6.0"
+$pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+$pkg_license=("Apache-2.0")
+$pkg_upstream_url="https://www.chef.sh"
+
+function Invoke-Install {
+  New-Item -ItemType directory -Path "${pkg_prefix}/lib"
+  Copy-Item -Path "$PLAN_CONTEXT/lib/scaffolding.ps1" -Destination "$pkg_prefix/lib/scaffolding.ps1"
+}
+
+$pkg_build_deps=@("core/chef-dk", "core/git")
```

Comment:
I need to follow up on this, but I believe that Chef Software would like all Chef Infra, Chef Workstation/DK, etc. packages released under the `chef` origin. `core/chef-dk` should be deprecated to avoid this confusion.  

We also [cannot declare a dependency on non-core packages](https://github.com/habitat-sh/core-plans/blob/master/CONTRIBUTING.md#dependencies) in the `core` origin.  The linux chef-scaffolding works around this by having the generated `plan.sh` declare the dependencies, rather than declare them directly.

Reply:
I agree, and core/chef-dk *will* be deprecated. However, there is no other current Windows build of chef-dk available.

As indicated in the parent comment, including these deps is necessary at the moment due to a bug in Habitat itself, since it does not respect the scaffolding's dependencies from `Load-Scaffolding`. But the intent is to remove this entirely in the future.

Reply:
I disagree.  We know that `core/chef-dk` should be deprecated, and if we depend on it now we're setting ourselves up for broken builds down the road.  

Let's file an issue on plan-build-ps1 to fix `Load-Scaffolding` (if there isn't one already) so that we can solve this in a way that is safe for our users and won't disrupt them down the road. Once that fix is in, we can revisit adding this to `core`.  In the mean time, this can be hosted under a different origin.

Reply:
I'd rather rip out the dependencies and force the downstream users to use them instead of switching origins. I'll do that now.

Reply:
Done! That should meet the criteria for merging this, since it now carries no dependencies of any kind.

Reply:
If this is chef scaffolding, why not host it in the chef origin?